oeps/oep-0001.rst

```diff
@@ -300,111 +366,84 @@ ReStructuredText [8] allows for rich markup that is relatively easy to read,
-+---------------+-------------------------------------------+
-| Status        | <Draft | Accepted | Deferred |            |
-|               | Rejected | Withdrawn | Final |            |
-|               | Replaced>                                 |
-+---------------+-------------------------------------------+
-| Type          | <Architecture | Best Practice |           |
-|               | Process>                                  |
-+---------------+-------------------------------------------+
-| Created       | <date created on, in YYYY-MM-DD format>   |
-+---------------+-------------------------------------------+
-| `Resolution`  | <links to any discussions where the final |
-|               | status was decided>                       |
-+---------------+-------------------------------------------+
-| `Replaces`    | <OEP number>                              |
-+---------------+-------------------------------------------+
-| `Replaced-By` | <OEP number>                              |
-+---------------+-------------------------------------------+
-| `References`  | <links to any other relevant discussions  |
-|               | or relevant related materials>            |
-+---------------+-------------------------------------------+
-
-The Author header lists the names, and optionally the email addresses, of all
-the authors/owners of the OEP. The format of the Author header value must be
-``Random J. User <address@dom.ain>`` if the email address is included, or
-``Random J. User`` if the address is not given. If there are multiple authors,
-their names and addresses should appear in a comma separated list.
-
-The Arbiter field is used to record who has the authority to make the final
-decision to approve or reject the OEP.
-
-The Type header specifies the type of OEP: Architecture, Best Practice, or
-Process.
-
-The Created header records the date that the pull request for the OEP was
-opened. It should be in YYYY-MM-DD format, e.g. 2016-04-21.
-
-OEPs can also have a Replaced-By header indicating that a OEP has been rendered
-obsolete by a later document; the value is the number of the OEP that replaces
-the current document. The newer OEP must have a Replaces header that contains
-the number of the OEP that it rendered obsolete.
++-----------------+-------------------------------------------+
+| OEP             | OEP-XXXX-<brief-type>-<brief-title>       |
++-----------------+-------------------------------------------+
+| Title           | <OEP title>                               |
++-----------------+-------------------------------------------+
+| Last Modified   | <date string, in YYYY-MM-DD format>       |
++-----------------+-------------------------------------------+
+| Authors         | <list of authors' real names and          |
+|                 | optionally, email addresses>              |
++-----------------+-------------------------------------------+
+| Arbiter         | <Arbiter's real name and email address>   |
```

Comment:
Given that an OEP can have multiple arbiters over time.  Do we want some sort of date associated with the aribter to indicate past aribters vs current arbiter?

Reply:
We don't have dates for authors either - as they can also change over time.
Can we rely on git for such details?

Reply:
For OEP-1, I went with providing a link to the PR for each arbitration.